micrometer/prometheus-kafka/src/test/java/io/quarkus/ts/openshift/micrometer/utils/AssertMetrics.java

```diff
@@ -0,0 +1,78 @@
+package io.quarkus.ts.openshift.micrometer.utils;
+
+import io.quarkus.ts.openshift.common.Command;
+
+import java.util.LinkedList;
+import java.util.List;
+import java.util.concurrent.TimeUnit;
+import java.util.function.Predicate;
+
+import static io.restassured.RestAssured.get;
+import static org.awaitility.Awaitility.await;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+public final class AssertMetrics {
+
+    private AssertMetrics() {
+
+    }
+
+    public static void thenMetricIsExposedInPrometheus(String name, Predicate<String> valueMatcher) throws Exception {
+        await().ignoreExceptions().atMost(5, TimeUnit.MINUTES).untilAsserted(() -> {
+            List<String> lines = new LinkedList<>();
+            new Command("oc", "exec", "-n", "openshift-user-workload-monitoring",
```

Comment:
Can you use OpenShiftClient instead of this? https://github.com/quarkus-qe/quarkus-openshift-test-suite#dependency-injection

Reply:
Didn't see fabric8 provided also this functionality, will use!